cmd/process-agent/collector.go

```diff
@@ -147,7 +200,8 @@ func (l *Collector) run(exit chan bool) error {
 			case <-heartbeat.C:
 				statsd.Client.Gauge("datadog.process.agent", 1, tags, 1) //nolint:errcheck
 			case <-queueSizeTicker.C:
-				updateQueueSize(len(processPayloads), len(podPayloads))
+				updateQueueBytes(processResults.Weight(), podResults.Weight())
```

Comment:
nit: if these two functions are always meant to be called together I think it makes sense to a have a single `updateQueue` function

Reply:
I think it could be a little confusing to call it with 4 ints.  I'd prefer to leave it if thats OK

Reply:
no problem to leave like this, but I actually thought of passing only 2 checkResult arguments.